### DISCUSSION TITLE
Add Errors to Dependency Dashboard

### DISCUSSION THREAD

**Discussion by adam-moss** [renovatebot/renovate#25821] [Suggest an Idea]
### Tell us more.

Would it be possible to enhance the dependency dashboard to include details of failures?

For example

https://github.com/renovatebot/renovate/blob/8cc11c3adbaa578676067f88f00bf24696bf7ead/lib/workers/repository/update/branch/index.ts#L708-L708 

Outputs to the log only but it would be useful to include in the dashboard for visibility.

**Comment by rarkins** [comment]
I thought it already included any warning level log messages
